src/dali/plugin/pair_converter/ccxt.py

```diff
@@ -52,13 +41,14 @@
 _TIME_GRANULARITY_IN_SECONDS: List[int] = [60, 300, 900, 3600, 14400, 86400]
 
 # Currently supported exchanges
+# NOTE coinbase currently (aug 2022) does not support the `fetchOHLCV()` method which is used here
 _BINANCE: str = "Binance.com"
-_CEX: str = "CEX.IO"
 _KRAKEN: str = "Kraken"
 _LIQUID: str = "Liquid"
+_GATE = "Gate"
```

Comment:
Can you add the type hint?

Reply:
I don't think we need them for string literals, right?